src/Mvc/test/WebSites/BasicWebSite/Controllers/NonNullableApiController.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+using Microsoft.AspNetCore.Mvc;
+
+namespace BasicWebSite.Controllers;
+
+#nullable enable
+
+[ApiController]
+[Route("api/NonNullable")]
+public class NonNullableApiController : ControllerBase
+{
+    // GET: api/<controller>
+    [HttpGet]
+    public ActionResult<string> Get(string language = "pt-br")
+    {
+        return language;
+    }
+}
+
+#nullable restore
```

Comment:
Nit: you don't need the `#nullable restore` since the `#nullable enable` is file scoped. You also have an extra `#nullable enable`.